Directions Example/ViewController.swift

```diff
@@ -18,16 +21,58 @@ class ViewController: UIViewController {
         
         mapView = MGLMapView(frame: view.bounds)
         mapView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
+        mapView.setCenter(ViewController.initialMapCenter, animated: false)
+        mapView.setZoomLevel(15, animated: false)
```

Comment:
Constant-ize as `ExampleStartingZoom`? Dealer's choice, it's the example.

Reply:
Or we could follow the pattern of the line above... `ViewController.initialZoom`